### DISCUSSION TITLE
Is there any pressing reason for us to upgrade from lighthouse 5.71 to version 6 or up?

### DISCUSSION THREAD

**Discussion by nicoloubser** [nuwave/lighthouse#2576] [General]
We have numerous systems we want to upgrade to PHP8.3 and Lumen 10. Due to specific reasons it will be very convenient to upgrade to lighthouse 5.72 only, and not higher. However, I find it difficult to determine whether this is a bad idea and whether there will be explicit benefits to upgrade lighthouse to version 6 or higher.

Are there for instance a spreadsheet comparing the different lighthouse systems security and speed wise, and until when they are supported? Or does anyone know from experience whether 5.72 is good enough and stable or maybe whether it will be a good choice for the next 2 years?

Thank you,

**Comment by spawnia** [comment]
As mentioned in https://github.com/nuwave/lighthouse?tab=readme-ov-file#versioning:

> Only the current major version receives new features and bugfixes.

The best source of information for things that changed are the [changelog](https://github.com/nuwave/lighthouse/blob/master/CHANGELOG.md) and [v5 to v6 upgrade guide](https://github.com/nuwave/lighthouse/blob/master/UPGRADE.md#v5-to-v6).
